include/functions.h

```diff
@@ -2534,8 +2534,8 @@ s32 func_8013E054(GlobalContext* globalCtx, struct_8013DF3C_arg1* arg1);
 s32 func_8013E07C(GlobalContext* globalCtx, struct_8013DF3C_arg1* arg1);
 s32 func_8013E0A4(GlobalContext* globalCtx, struct_8013DF3C_arg1* arg1);
 void SubS_ChangeAnimationBySpeedInfo(SkelAnime* skelAnime, AnimationSpeedInfo* animations, s32 nextIndex, s32* curIndex);
-s32 func_8013E2D4(Actor* actor, s16 arg1, s16 arg2, s32 arg3);
-s32 func_8013E3B8(Actor* actor, s16 cutscenes[], s16 len);
+s32 SubS_StartActorCutscene(Actor* actor, s16 nextCutscene, s16 curCutscene, s32 type);
+s32 SubS_FillCutscenesList(Actor* actor, s16 cutscenes[], s16 cutscenesLen);
```

Comment:
```suggestion
s32 SubS_FillCutscenesList(Actor* actor, s16 cutscenes[], s16 numCutscenes);
```
Cutscene Length makes me think of the number of frames a cutscene would last. I think `Num` is more common when it's related to count. Would change in the actual function to

Reply:
Also updated the limbRotTable function to match this style.